test/middleware_test.rb

```diff
@@ -202,7 +229,7 @@ def test_cache_miss_and_store_with_gzip_support
     ).once
 
     env = Rack::MockRequest.env_for("http://example.com/index.html")
-    env['HTTP_ACCEPT_ENCODING'] = 'deflate, gzip'
+    env['HTTP_ACCEPT_ENCODING'] = 'br'
```

Comment:
for completeness, this should be `gzip, deflate, br` to match common `accept-encoding` patterns